docs/Spark/configuration/configuration.md

```diff
@@ -65,6 +101,12 @@ In the below image `Config.num_top_customers` is fetching the integer value defi
 
 ![Config Limit Example](img/config-pipeline-limit-eg-scala-python.png)
 
+#### Jinja Config
+
+In the below image `{{num_top_customers}}` is fetching the integer value defined in configurations.
+
+![Config Limit Example](img/config-pipeline-limit-eg-jinja.png)
```

Comment:
Should give an example on the basis of property field datatypes wise as well ??
Like, in case the gem property datatype is
`SColumnExpression` -> `lit("{{a.b.c}}")`
`SString` -> `{{ a.b.c }}`
`String`-> `{{ a.b.c }}`

Thoughts ??

Reply:
Done.